docker/pulsar/Dockerfile

```diff
@@ -36,6 +34,25 @@ COPY scripts/set_python_version.sh /pulsar/bin
 COPY scripts/install-pulsar-client-27.sh /pulsar/bin
 COPY scripts/install-pulsar-client-35.sh /pulsar/bin
 
+
+### Create 2nd stage from OpenJDK image
+### and add Python dependencies (for Pulsar functions)
+
+FROM openjdk:8-jdk-slim
+COPY --from=pulsar /pulsar /pulsar
+
+# Install some utilities
+RUN apt-get update \
+     && apt-get install -y netcat dnsutils \
+                 python2.7 python-setuptools \
+                 python3 python3-setuptools \
+     && apt-get clean \
+     && rm -rf /var/lib/apt/lists/*
+
+RUN easy_install pip
+RUN pip install kazoo pyyaml
+RUN easy_install3 pip
```

Comment:
need pip3 install kazoo pyyaml

Reply:
Do we need both for py2 and py3?

Reply:
I am in favor of dropping python2. I think everything works with py3 except the dashboard.